e2e/cypress/integration/messaging/message_reply_gm_spec.js

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @messaging
+
+import {getRandomId} from '../../utils';
+
+describe('MM-T470 Reply in existing GM', () => {
```

Comment:
We can remove the TM4J key here as it's already added in the test case below

```suggestion
describe('Reply in existing GM', () => {
```